scripts/dask/README.md

```diff
@@ -0,0 +1,45 @@
+# Dask scripts for multi-GPU environments
+
+This directory contains tools for configuring environments for single-node or
+multi-node, multi-gpu (SNMG or MNMG) Dask-based cugraph runs, currently
+consisting of shell and python scripts.
+
+Users should also consult the multi-GPU utilities in the
+`python/cugraph/cugraph/testing/mg_utils.py` module, specifically the
+`start_dask_client()` function, to see how to create `client` and `cluster`
+instances in Python code to access the corresponding Dask processes created by
+the tools here.
+
+
+### run-dask-process.sh
+
+ This script is used to start the Dask scheduler and workers as needed.
+
+ To start a scheduler and workers on a node, run it like this:
+ ```
+ bash$ run-dask-process.sh scheduler workers
```

Comment:
I am not sure if we should add more details in the README regarding the different communication framework supported like `--tcp`, `--ucx`, `--ucxib`

Reply:
I added those details, thanks.